<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>1092</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1165</width>
    <height>1092</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1151</width>
      <height>1041</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
</string>
       </property>
       <property name="text">
        <string>Количество отрабонных часов:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="textEdit_7">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
align-items: center; /* Выравнивание по вертикали */
background-color: #F5F5DC;</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Сумма до вычета НДФЛ:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="textEdit">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
align-items: center; /* Выравнивание по вертикали */
background-color: #F5F5DC;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
height: 100px;
     
</string>
       </property>
       <property name="text">
        <string>Вид Зарплаты:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="textEdit_8">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
align-items: center; /* Выравнивание по вертикали */
background-color: #F5F5DC;</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>18</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;

display: inline-block;
text-align: center;

padding: 10px 20px;
margin-left: 200px;
margin-right: 200px;
border: 30px;
border-radius: 20px; /* Скругление углов */
background-color: #007bff; /* Цвет фона */
color: #fff; /* Цвет текста */
cursor: pointer;
</string>
       </property>
       <property name="text">
        <string>Paссчитать зарплату</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>18</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;MS Shell Dlg 2&quot;;

display: inline-block;
text-align: center;

padding: 10px 20px;
margin-left: 200px;
margin-right: 200px;
border: 30px;
border-radius: 20px; /* Скругление углов */
background-color: #007bff; /* Цвет фона */
color: #fff; /* Цвет текста */
cursor: pointer;
</string>
       </property>
       <property name="text">
        <string>Выбрать</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Сумма на руки:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Коэфициент за сверхурочные:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
height: 100px;
     
</string>
       </property>
       <property name="text">
        <string>Месяц:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <property name="styleSheet">
        <string notr="true">height: 100px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
align-items: center; /* Выравнивание по вертикали */
background-color: #F5F5DC;</string>
       </property>
       <item>
        <property name="text">
         <string>                январь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                февраль</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                март</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                апрель</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                май</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                июнь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                июль</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                август</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                сентябрь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                октябрь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                ноябрь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                декабрь</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Отработанные дни</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Сумма НДФЛ, 13%:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QTextEdit" name="textEdit_4">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
align-items: center; /* Выравнивание по вертикали */
background-color: #F5F5DC;
height: 10px;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="styleSheet">
        <string notr="true">height: 100px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
align-items: center; /* Выравнивание по вертикали */
background-color: #F5F5DC;</string>
       </property>
       <item>
        <property name="text">
         <string>                ПОЧАСОВАЯ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                ПОМЕСЯЧНАЯ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>                ПРЕМИАЛЬНАЯ</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QTextEdit" name="textEdit_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
align-items: center; /* Выравнивание по вертикали */
background-color: #F5F5DC;</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QTextEdit" name="textEdit_3">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
align-items: center; /* Выравнивание по вертикали */
background-color: #F5F5DC;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Размер почасовой оплаты: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Оклад</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QTextEdit" name="textEdit_5">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
align-items: center; /* Выравнивание по вертикали */
background-color: #F5F5DC;</string>
       </property>
      </widget>
     </item>
     <item row="11" column="3">
      <widget class="QTextEdit" name="textEdit_6">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
align-items: center; /* Выравнивание по вертикали */
background-color: #F5F5DC;</string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>label_1</zorder>
    <zorder>comboBox</zorder>
    <zorder>pushButton</zorder>
    <zorder>label_9</zorder>
    <zorder>textEdit_8</zorder>
    <zorder>label_8</zorder>
    <zorder>comboBox_2</zorder>
    <zorder>textEdit_7</zorder>
    <zorder>label</zorder>
    <zorder>textEdit</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>textEdit_2</zorder>
    <zorder>label_4</zorder>
    <zorder>textEdit_3</zorder>
    <zorder>label_5</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>textEdit_4</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>textEdit_6</zorder>
    <zorder>textEdit_5</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
